tensorflow/core/common_runtime/gpu/gpu_cudamallocasync_allocator.cc

```diff
@@ -111,6 +111,32 @@ GpuCudaMallocAsyncAllocator::GpuCudaMallocAsyncAllocator(
         pool_, CU_MEMPOOL_ATTR_REUSE_ALLOW_INTERNAL_DEPENDENCIES, &disable);
   }
 
+  // Set read/write access to all GPUs.
+  for(int i = 0; i < all_pools_.size(); ++i) {
+    // Set the current pool access to the previous GPUs.
+    CUmemAccessDesc map;
+    map.flags = CU_MEM_ACCESS_FLAGS_PROT_READWRITE;
+    map.location.id = all_ids_[i].value();
+
+    map.location.type = CU_MEM_LOCATION_TYPE_DEVICE;
+    if (auto status = cuMemPoolSetAccess(pool_, &map, 1)) {
+      pool_ = nullptr;
+      LOG(FATAL) << "could not set access to the pool : "
```

Comment:
Prefer not to use `LOG(FATAL)`, or if there is no other way, please mark it with `// Crash OK.` like above.

Reply:
Done.